read.go

```diff
@@ -39,6 +40,9 @@ func readVR(r dicomio.Reader, isImplicit bool, t tag.Tag) (string, error) {
 		}
 		return tag.UNKNOWN, nil
 	}
+	if t == tag.Item {  //http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_7.5.html
+		return "na", nil
+	}
```

Comment:
I found this error during testing and I'm not quite sure what the best fix is. Based on this web page: http://dicom.nema.org/dicom/2013/output/chtml/part05/sect_7.5.html
It says that Sequence Items don't have explicit VRs written and the 4 bytes immediately after the tag is the value length as a UINT32

Reply:
Thanks for raising this! Do you have an example DICOM that has an issue?

I think all we need to do is just ensure that elements are read with an implicit transfer syntax if they are an item.

e.g something like

if t == tag.Item {
    force_implicit = true
}

I can send a quick PR to address this, so this one can be mostly focued on the write specific logic--what I'm worried about is why we haven't seen an issue yet in previous testing. (Maybe because we had files encoded in implicit transfer syntax for the sequence testing?)

Either way, if you have a problematic file we can add to the set of test files, that would be great!

Reply:
See #122, I __think__ this will solve this, but some test data is essential (that was breaking before)

Reply:
Sounds good. I agree it may be better to address this in another PR. I've taken a look at the changes in #122 and made some comments. I'll try and attach a test file that demonstrated this issue. The problem is actually very subtle because no error gets raised, the parsing just ends prematurely after the first sequence and it thinks it has read the entire file.